### PR TITLE
automate case OCP-24222

### PR DESCRIPTION
**Pull request by lihongyan1**
Automate case
https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-24222
v3 job
https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/94064/console
Depend on PR:
https://github.com/openshift/cucushift/pull/7784

### CODE REVIEW

**Comment by lihongyan1** [comment]
@juzhao @yapei Please help review

**Comment by yapei** [comment]
look good to me

File: lib/rules/web/admin_console/4.1/monitoring_metrics.xyaml [hunk 1]
```diff
@@ -127,10 +127,37 @@ check_metric_query_result:
...
     selector:
-      xpath: //div[@class='odc-metrics-chart' ]//div[@class='query-browser__zoom']
+      xpath: //div[@class='query-browser__zoom']
     timeout: 120
 check_metric_query_table_text:
   element:
     selector:
       xpath: //table[@aria-label='query results table']//td[contains(text(),'<table_text>')]
     timeout: 120
+
```
Comment (juzhao) [inline lib/rules/web/admin_console/4.1/monitoring_metrics.xyaml:137]: Line 137 - 163 is no needed for 4.1, since the case is for 4.2 and above

**Comment by juzhao** [comment]
runner v3 job:
https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/97897/console

**Pull request merged**